docs/theming/high-contrast-mode.md

```diff
@@ -0,0 +1,245 @@
+
+import AlwaysHighContrastMode from '@site/static/usage/v8/theming/always-high-contrast-mode/index.md';
+
+<AlwaysHighContrastMode />
+
+### System
+
+The system approach to enabling high contrast mode involves checking the system settings for the user's preferred contrast. This is the default when starting a new Ionic Framework app. Importing the following stylesheets in the appropriate file will automatically retrieve the user's preference from the system settings and apply the high contrast theme when high contrast is preferred.
+
+The following example shows how to include both the high contrast light theme as well as the high contrast dark theme. The system's dark mode preference will be checked to show either the light or dark variant of the high contrast theme.
+
+<Tabs groupId="framework" defaultValue="angular" values={[{ value: 'angular', label: 'Angular' }, { value: 'javascript', label: 'Javascript' }, { value: 'react', label: 'React' }, { value: 'vue', label: 'Vue' }]}>
+
+<TabItem value="angular">
+
+```css
+@import '@ionic/angular/css/themes/high-contrast.system.css';
+@import '@ionic/angular/css/themes/high-contrast-dark.system.css';
+```
+
+</TabItem>
+<TabItem value="javascript">
+
+```ts
+import '@ionic/core/css/themes/high-contrast.system.css';
+import '@ionic/core/css/themes/high-contrast-dark.system.css';
+```
+
+</TabItem>
+<TabItem value="react">
+
+```tsx
+import '@ionic/react/css/themes/high-contrast.system.css';
+import '@ionic/react/css/themes/high-contrast-dark.system.css';
+```
+
+</TabItem>
+<TabItem value="vue">
+
+```ts
+import '@ionic/vue/css/themes/high-contrast.system.css';
+import '@ionic/vue/css/themes/high-contrast-dark.system.css';
+```
+
+</TabItem>
+
+</Tabs>
+
+This approach activates the high contrast theme when the [CSS media query for `prefers-contrast`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-contrast) is `more`. The `prefers-contrast` media query is supported by [all modern browsers](https://caniuse.com/?search=prefers-contrast). If support for an older browser is required, we recommend using the [class](#class) approach.
+
+The following example uses the system settings to decide when to show high contrast mode.
+
+:::info
+Not sure how to change the system settings? Here's how to enable high contrast mode on [Windows 11](hhttps://support.microsoft.com/en-us/windows/turn-high-contrast-mode-on-or-off-in-windows-909e9d89-a0f9-a3a9-b993-7a6dcee85025) and on [macOS](https://support.apple.com/guide/mac-help/change-display-settings-for-accessibility-unac089/mac).
+:::
+
+import SystemHighContrastMode from '@site/static/usage/v8/theming/system-high-contrast-mode/index.md';
+
+<SystemHighContrastMode />
+
+:::caution
+The high contrast light theme must be imported after [core.css](../layout/global-stylesheets#corecss), and the
+high contrast dark theme must be imported after `dark.system.css`. Otherwise, the standard contrast theme will take priority.
+:::
+
+### Class
```

Comment:
```suggestion
### CSS Class
```

This change would also impact some relative links in the doc, so open to not accepting it. I'm just wondering after reading these and pretending I know nothing, would I know what "Class" is referring to in the side nav?

Reply:
Good point. I followed what was done on the dark mode docs, so I made the change there too: https://github.com/ionic-team/ionic-docs/pull/3431/commits/225dc62d4ac724ef797ef341797fc55cdb4f1a6d